src/fetch_modules.js

```diff
@@ -3,9 +3,9 @@ export default function(nodeVersion) {
     return $.Deferred().resolve(window.wtModuleList)
   }
 
-  const url = nodeVersion == '4' ?
-    'https://auth0-internal.us.webtask.io/canirequire' :
-    'https://auth0-internal.us8.webtask.io/canirequire'
+  const url = nodeVersion == '8' ?
```

Comment:
Can we use a dictionary for mapping the versions to URLs?
I'm thinking it might be easier should we need support more than two versions at a time in near future (Node 14).